NuGet.ruleset

```diff
@@ -1,236 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="NuGet Code Analysis Rules" ToolsVersion="14.0">
-  <IncludeAll Action="Warning" />
-  <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
```

Comment:
Included in minimum required ruleset For the entire set to apply there's a lot of work to fix all the existing issues. Will work to elevate code analysis level later as part of NuGet/Home#4722 epic.